cdap-ui/app/directives/dag-plus/my-dag-ctrl.js

```diff
@@ -423,6 +442,25 @@ angular.module(PKG.name + '.commons')
     }
 
     function addConnections() {
+      const addConnectionToErrorsAlerts = (conn, sourceNode, targetNode, connObj) => {
+        if (vm.shouldShowErrorsPort(sourceNode) && targetNode.type === 'errortransform') {
```

Comment:
If we flip the condition order then we would check if we need to show error ports only for error transform plugins right? Slightly better?